mq/environments/ci/tasks/4-smoke-tests-mq.yaml

```diff
@@ -239,23 +230,14 @@ spec:
     - name: cleanup
       image: $(params.tools-image)
       workingDir: $(params.source-dir)
-      env:
-        - name: TLS_SECRET_NAME
-          valueFrom:
-            configMapKeyRef:
-              name: ibmcloud-config
-              key: TLS_SECRET_NAME
-              optional: true
-        - name: INGRESS_SUBDOMAIN
-          valueFrom:
-            configMapKeyRef:
-              name: ibmcloud-config
-              key: INGRESS_SUBDOMAIN
-              optional: true
       script: |
           #!/usr/bin/env bash
           set -ex
 
           APP_NAMESPACE="$(params.app-namespace)"
 
           kubectl delete -n ${APP_NAMESPACE} -f ./release.yaml
+
+          if [[ "$(params.storage)" == "true" ]]; then
+            kubectl delete --all pvc -n ${APP_NAMESPACE}
+          fi
```

Comment:
@phemankita, instead of deleting all pvcs in the `ci` namespace, we should update it to delete the specific one(s) created by the durable qm.

Reply:
Let's look to fix this in a future PR.